common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -311,6 +310,8 @@ def gate_sequence_if_it_is_a_timed_exam_and_contains_content_type_gated_problems
         Note that this will break compatability with using sequences outside of edx-platform
         but we are ok with this for now
         """
+        from django.contrib.auth.models import User
+
```

Comment:
This `User` import was recently added in https://github.com/edx/edx-platform/commit/95811992ca4a3c0fcf1e6584b67c905f4ff36b39. Need to move it inside the method otherwise `xmodule_assets common/static/xmodule` fails.